README.md

```diff
@@ -82,7 +87,3 @@ Resources:
 - [ZenHub Board](https://app.zenhub.com/workspaces/wildfire-predictive-services-5e321393e038fba5bbe203b8/board?repos=235861506,237125626,237125691)
 - [PEP8](https://github.com/python/peps/blob/master/pep-0008.txt) and [PEP20](https://github.com/python/peps/blob/master/pep-0020.txt) coding conventions, but with 110 character line breaks
 - [Code of Conduct](https://github.com/bcgov/wps/blob/master/CONDUCT.md)
-
```

Comment:
Don't drop this - we'll still keep using sonarcloud, just not for coverage.

Reply:
What do we want to keep sonarcloud for?

Reply:
Does static analysis. Catches security issues, code smells etc. etc.